source/_patterns/components/04-toast-notifications/00-toast-notification.hbs

```diff
@@ -0,0 +1,14 @@
+<div class="pf-toast alert {{ alertClass }} {{#if alertDismissible}}alert-dismissible{{/if}}" role="alert">
+{{#if alertDismissible}}
+  <button type="button" class="close" data-dismiss="alert" aria-label="Close">
```

Comment:
I'm using this button in the context of Alerts: http://v4-alpha.getbootstrap.com/components/alerts/#dismissing

My changes were to add the pficon-close and remove the &times; , because we use an icon instead of this character.